readme.md

```diff
@@ -351,6 +376,17 @@ class Dog extends Animal {
 
 -------
 
+### Further Reading: Class vs. Prototypical Inheritance
+
+When reading about object oriented programming, you'll come across two popular paradigms: **class** vs. **prototypical**. Examples of class-based programming languages are Ruby and Java. Even though we are using something called a "class" in Javascript, however, Javascript employs prototypical inheritance.
+
+Classes became a part of Javascript with the implementation of ES6. Under the hood, however, classes are using constructor functions and prototypes, two fundamental tools that were used to implement OOJS prior to ES6.
+
+We will not be explicitly covering prototypical inheritance in this course. If you are interested in learning more, however, check out these resources...
```

Comment:
Point here is to clarify that this is not true classical inheritance and that actually prototypical inheritance is what's happening under the hood.

I'll reword it. Didn't come out right the first time.